src/main/java/school/lemon/changerequest/java/introduction/pr1/ConsoleCalculator.java

```diff
@@ -1,9 +1,98 @@
 package school.lemon.changerequest.java.introduction.pr1;
 
 
+import java.util.Scanner;
+
 public class ConsoleCalculator {
 
     public static void main(String[] args) {
+        System.out.println("Console calculator:");
+        System.out.println("Enter 'add' to perform addition.");
+        System.out.println("Enter 'sub' to perform subtraction.");
+        System.out.println("Enter 'mul' to perform multiplication.");
+        System.out.println("Enter 'div' to perform division.");
+        System.out.println("Enter 'exit' to exit.");
+        System.out.println("Enter 'help' to see help message.");
+        boolean start = true;
+        while (start) {
+            System.out.println("Make your choice.");
+            Scanner sc = new Scanner(System.in);
+            String choice = sc.nextLine();
+            choice.toLowerCase();
+            switch (choice) {
+                case "add":
+                    add();
+                    break;
+                case "sub":
+                    sub();
+                    break;
+                case "mul":
+                    mul();
+                    break;
+                case "div":
+                    div();
+                    break;
+                case "exit":
+                    exit();
+                    break;
+                case "help":
+                    help();
+                    break;
+                default:
+            }
+        }
+
+
+    }
+
+    private static void help() {
+    }
+
+    private static void exit() {
+        System.out.println("Bye-bye");
+        System.exit(0);
+    }
+
+    private static void div() {
```

Comment:
There are too many duplicated code... Reading two integers for example. Maybe you can get rid of it?